Negotiation/FormatNegotiator.php

```diff
@@ -18,26 +18,15 @@
 use Symfony\Component\HttpFoundation\RequestMatcherInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
 
+/**
+ * @author Ener-Getick <energetick.guigui@gmail.com>
+ */
 class FormatNegotiator extends BaseNegotiator
 {
-    /**
-     * @var array
-     */
     private $map = [];
-
-    /**
-     * @var RequestStack
-     */
     private $requestStack;
```

Comment:
In Symfony (and in my own code), we only document it when the constructor does not allow inferring it. This avoids having to maintain the phpdoc for them (which could become outdated)